tools/actions/create-release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -xe
+
+RELEASE_DATE=$1
+
+git node release --prepare --skipBranchDiff
+# We use it to not specify the branch name as it changes based on
+# the commit list (semver-minor/semver-patch)
+git config push.default current
+echo "/## $RELEASE_DATE/,/^<a id=/{ if (!/^<a id=/) print }" > temp.awk
+awk -f temp.awk doc/changelogs/CHANGELOG_V23.md > pr-body.md
```

Comment:
```suggestion
MATCHER="/## $RELEASE_DATE/,/^<a id=/{ if (!/^<a id=/) print }"
awk $MATCHER doc/changelogs/CHANGELOG_V23.md > pr-body.md
```
Can't this be passed directly?

If not
```suggestion
MATCHER="/## $RELEASE_DATE/,/^<a id=/{ if (!/^<a id=/) print }"
awk -f <(echo $MATCHER) doc/changelogs/CHANGELOG_V23.md > pr-body.md
```

IMO this is better than creating a file and deleting it right after.

Reply:
Why this is better?

Reply:
This way, in case of a failure in the awk command, no artifacts are left over. Additionally, it prevents a needless call to write to the file system

Reply:
I agree, relying on FS like that means it’s theoretically possible some other process mutates the temp files in between the two commands. The same applies for `pr-body.md` btw, we don’t need it, we can simply chain the two commands.